.github/workflows/ci.yaml

```diff
@@ -18,5 +18,7 @@ jobs:
     - uses: actions/setup-go@v2.1.3
       with:
         go-version: 1.16.x
+    - name: Ensure generated code is committed
+      run: make generate && make manifests && [[ -n $(git status -s) ]] && exit 1
```

Comment:
I have a slight preference for `git diff --exit-code` as it will show the diff and exit 1

Reply:
```suggestion
      run: make generate manifests && git diff --exit-code
```